website/docs/r/nsxt_edgegateway.html.markdown

```diff
@@ -353,8 +353,10 @@ can be used to lookup ID by name.
   support IP Spaces*.
 * `read_limit_unused_ip_count` - (Optional, *v3.13+*) Sets a limit of IPs to count for
   `used_ip_count` and `unused_ip_count` attributes to avoid exhausting compute resource while
-  counting IPs in large IPv6 subnets. Defaults to `1000000`, update is a no-op, but will affect
-  newly read data.
+  counting IPs in large IPv6 subnets. It does not affect operation of Edge Gateway configuration,
```

Comment:
Sorry for a late thought. I was re-reading the PR. I think it would be clearer if this field was named (if it's not too late to change, of course):
`ip_count_read_limit`

Witht this suggestion I'm trying to address two things:
1. Right now it says "..._unused_ip_count", but description says it's actual for both
  `used_ip_count` and `unused_ip_count` attributes.
2. Putting the "_read_limit" at the end makes it clearer, IMHO.

Additional thought: is "read" really precise here? It sounds like API GET call limit. Maybe just name the field `ip_count_limit`?

Reply:
> Sorry for a late thought. I was re-reading the PR. I think it would be clearer if this field was named (if it's not too late to change, of course):
> `ip_count_read_limit`

Not an issue at all. The heavy lifting is in SDK anyway.

> Additional thought: is "read" really precise here? It sounds like API GET call limit. Maybe just name the field `ip_count_limit`?

What happens is that when there is a huge subnet, VCD API does not return IP count for unused IPs anymore (with small subnets it does). Then, to count them our code has to iterate over all IPs in defined subnets (this is what can overload compute with large subnets). So I would say this is our read that can exhaust resources. It does return the `used_ips` as it has a dedicated endpoint that tells where each IP is used.

`ip_count_limit` sounds too generic for me, but `ip_count_read_limit` sounds OK to me

Reply:
`ip_count_read_limit` it is then! :)

Reply:
Changed. Test `TestAccVcdIpv6SupportLargeSubnet` still passes.